llama-index-integrations/indices/llama-index-indices-managed-google/llama_index/indices/managed/google/base.py

```diff
@@ -25,7 +25,6 @@
 from llama_index.core.indices.base_retriever import BaseRetriever
 from llama_index.core.indices.managed.base import BaseManagedIndex
 from llama_index.core.indices.query.base import BaseQueryEngine
-from llama_index.core.indices.service_context import ServiceContext
```

Comment:
@masci just checking if I ripped this out correctly. Its already ripped from core, so I think I did this correctly?